requirements.txt

```diff
@@ -22,6 +22,7 @@ decorator==4.0.11         # via datadog
 docutils==0.13.1          # via botocore
 ecdsa==0.13               # via python-jose
 enum34==1.1.6             # via cryptography, h2
+factory-boy==2.8.1
```

Comment:
I don't see where this is needed (outside of test-requirements.txt's realm)

Reply:
I had to add it in order to get `setup.py devel` to work for my local environment. I thought it might be an oversight since I think we all had installed it separately.